server/src/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -3085,9 +3085,10 @@ public GuestVlan dedicateGuestVlanRange(DedicateGuestVlanRangeCmd cmd) {
         // Verify account is valid
         Account vlanOwner = null;
         if (projectId != null) {
-            if (accountName != null) {
-                throw new InvalidParameterValueException("accountName and projectId are mutually exclusive");
-            }
+            //accountName and projectId are mutually exclusive
```

Comment:
@koushik-das : If we go with the scope field, we have to revert that change and have to make required depending on the scope we select. Like shown in the below snapshot.

<img width="420" alt="screen shot 2017-02-22 at 6 14 19 pm" src="https://cloud.githubusercontent.com/assets/12583725/23212092/1ba1cd22-f92b-11e6-83ba-72dab04dc497.png">